app/views/components/buttons/remove_button.scala.html

```diff
@@ -18,4 +18,4 @@
 
 @(value: Option[Any] = None, label: String = "site.remove", buttonClass: String = "button--secondary")(implicit messages: Messages)
 
-<button id="remove" class="@buttonClass" name="@Remove.toString" @value.map{v => value="@{v.toString}"}>@messages(label)</button>
\ No newline at end of file
+<button class="remove @buttonClass" name="@Remove.toString" @value.map{v => value="@{v.toString}"}>@messages(label)</button>
```

Comment:
What is the "remove" css class?  And should this button not have an id?

Reply:
Those buttons are used multiple time on site and break uniqueness of id there - remove is just access class same as id but its legal to have multiple tags to share it